framework/src/modules/chain/transport/transport.js

```diff
@@ -247,11 +247,11 @@ class Transport {
 			);
 		}
 
-		// TODO: endpoint should be protected before
-		if (this.synchronizer.isActive) {
+		// Should ignore received block if syncing
+		if (this.synchronizer.isActive()) {
```

Comment:
`isActive` is getter so it shouldn't need `()`

Reply:
Which now makes me think we shouldn't include verbs in getter functions

Reply:
ah yea that's true

Reply:
So or either `synchronizer.active` accesses the `active` variable, or `synchronizer.isActive()` IMHO

Reply:
variable which we don't want to be changed from outside, so i think it make sense to use method?

Reply:
Yeah, `isSomething` should be function IMO

Reply:
But then we can declare a getter like:
```javascript
get active() {
    return this._active;
}
```

This way you set `active` to "private" and IDEs won't show it as a public property when accessing the synchronizer.

Reply:
I've removed the `()` as refactoring synchronizer is out of scope of this PR